README.md

```diff
@@ -1,49 +1,30 @@
-## contexture-react
-`contexture-react` is a suite of react components for building contexture interfaces
+# contexture-react
 
+React components for building contexture interfaces https://smartprocure.github.io/contexture-react
 
 [![CircleCI](https://circleci.com/gh/smartprocure/contexture-react.svg?style=svg)](https://circleci.com/gh/smartprocure/contexture-react)
 [![Greenkeeper badge](https://badges.greenkeeper.io/smartprocure/contexture-react.svg)](https://greenkeeper.io/)
```

Comment:
not a comment on this PR, but we really should remove these broken greenkeeper badges (or, fix them and actually start using greenkeeper 😂)

Reply:
Yup